### DISCUSSION TITLE
v4 봇 채팅 읽음 관련

### DISCUSSION THREAD

**Discussion by difmwng3** [storycraft/node-kakao#435] [Q&A]
봇이랑 1대1 대화방에서

봇은 메세지를 읽었다고 표시되지않던데 ? (1대1 채팅방에서 제가 메세지를보내면 1(읽음표시)이 남는데  봇이 응답하면 0이 되어야 되는데 1로 표시됨)

즉 패킷으로 샌딩하니까 읽었다고 표시가 안됨.

봇으 해당 메세지를 읽었다! 라고 보내는 함수는없나요 ?

**Comment by storycraft** [comment]
TalkChannel객체의 markRead 메서드를 사용해보세요
